### ISSUE TITLE
Label offsets for MarkerOverlay are ignored

### ISSUE THREAD

**Issue by KaiKikuchi** [TeamJM/journeymap-api#26]
Currently, the `TextProperties.setOffsetX(int)` and `TextProperties.setOffsetY(int)` methods do not work for MarkerOverlays. Those properties are simply ignored. The marker is still shown on the fullscreen map and minimap like they have never been set. Other options like the background color work properly though.

This makes it harder to see the markers' icon and read the markers' label.

Mods used:
- journeymap-1.19.4-5.9.5-fabric.jar
- fabric-api-0.76.0+1.19.4.jar
- JM-API version: 1.19.4-1.9-fabric-SNAPSHOT

**Comment by mysticdrew** [comment]
MC, JM, and JM-API versions?

**Comment by KaiKikuchi** [comment]
> MC, JM, and JM-API versions?

Sorry I edited my comment with the info just after I posted my issue.

**Comment by mysticdrew** [comment]
I've moved this issue to the journeymap issue tracker since it an issue inside Journeymap itself. I will close this when the fix is release. 

Can you share your marker creation code? Github repo or gist will be fine too. 

Thanks!

**Comment by KaiKikuchi** [comment]
I made this test code to debug the issue:
```java
@Override
public void onEvent(ClientEvent e) {
  try {
    switch(e.type) {
    case MAPPING_STARTED:
      if (jmAPI.playerAccepts(MODID, DisplayType.Marker)) {
        ResourceLocation sprites = new ResourceLocation("journeymap:ui/img/marker-player-bg-32.png");
        MapImage icon = new MapImage(sprites, 0, 0, 32, 32, 0xFF0000, 1f);
        icon.setDisplayHeight(32);
        icon.setDisplayWidth(32);
        icon.setAnchorX(15.5);
        icon.setAnchorY(15.5);
        icon.setRotation(0);
        
        MarkerOverlay overlay = new MarkerOverlay(MODID, "test", new BlockPos(0, 64, 0), icon);
        overlay.setLabel("Test Label");
        overlay.setDimension(Level.OVERWORLD);
        
        // Setting offsetY to 0, 10 or 100 will show the label centered to the icon point.
        // The background color is set correctly to white though!
        overlay.getTextProperties().setBackgroundColor(0xFFFFFF).setOffsetY(100);
        
        jmAPI.show(overlay);
      }
      break;
    case MAPPING_STOPPED:
      jmAPI.removeAll(MODID);
      break;
    default:
    }
  } catch (Throwable t) {
    t.printStackTrace();
  }
}
```

Result:
![image](https://user-images.githubusercontent.com/8842403/231873071-64a8ed30-7a50-453c-9498-edf912d88b03.png)


Thank you for the quick answer!

**Comment by mysticdrew** [comment]
thanks

**Comment by mysticdrew** [comment]
This will be fixed in Journeymap 5.9.7. 
It was a simple case of `xShift != 0 && yShift != 0` when it should be `xShift != 0 || yShift != 0`.

**Comment by mysticdrew** [comment]
5.9.7 is on curseforge and modrinth with the fix - no api update is needed
